### PR TITLE
Add missing Russian translation

### PR DESCRIPTION
**Pull request by Evalle**
Signed-off-by: evalle <shmarnev@gmail.com>

### CODE REVIEW

File: doc/ru/weechat_quickstart.ru.asciidoc [hunk 1]
```diff
@@ -325,20 +323,17 @@ You can add an IRC server with `/server` command, for example:
...
 /script install iset.pl
 ----
 
-// TRANSLATION MISSING
-See `/help script` for more info.
+Для получения дополнительной информации введите команду `/help script`.
 
-// TRANSLATION MISSING
-A list of scripts is available in WeeChat with `/script` or at this URL:
+Список скриптов, доступных в WeeChat можно просмотреть введя команду `/script` или на сайте:
```
Comment (oakkitten) [inline doc/ru/weechat_quickstart.ru.asciidoc]: пунктуация

Comment (flashcode) [inline doc/ru/weechat_quickstart.ru.asciidoc]: oakkitten: are you reporting a typo in the commit? (if possible please mention in English the problem in the commit, thanks)

Comment (Evalle) [inline doc/ru/weechat_quickstart.ru.asciidoc]: Hello, 
@oakkitten thank you for your message.
@flashcode I have a missed commas in my commit, I will fix it in an hour.

**Comment by flashcode** [comment]
Merged.

**Pull request closed without merging**